vendor.list

```diff
@@ -4,6 +4,7 @@ github.com/cenkalti/backoff
 github.com/davecgh/go-spew
 github.com/dgrijalva/jwt-go v3.0.0
 github.com/dustin/go-humanize
+github.com/eclipse/paho.mqtt.golang
```

Comment:
Can you rebase your changes? This file is no longer in use in master. Kapacitor has switched over to using the new golang dep tool. Info on how to use it is in the CONTRIBUTING.md file.